install_old_edge_webdriver.txt

```diff
@@ -0,0 +1,3 @@
+Run this command in powershell:
```

Comment:
remove this file or add it in the readme file, as a SetUp browser drivers, which would actually be nice to have for all of them.